tests/components/cert_expiry/test_init.py

```diff
@@ -107,3 +108,26 @@ async def test_unload_config_entry(mock_now, hass):
     await hass.async_block_till_done()
     state = hass.states.get("sensor.cert_expiry_timestamp_example_com")
     assert state is None
+
+
+async def test_delay_load_during_startup(hass):
+    """Test loading a new config entry during runtime."""
+    hass.state = CoreState.starting
```

Comment:
This state means that the core is running.

Reply:
Init this test with `CoreState.not_running` instead?